IOS10-HW12-Maria Glushtsova/Sources/ViewController.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  ViewController.swift
+//  IOS10-HW12-Maria Glushtsova
+//
+//  Created by Admin on 21.06.23.
+//
+
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+    
+    // MARK: - Outlets
+    
+    var timer = Timer()
+    var timeToShow = 10
+    var workTimeRemaining = 10
+    var breakTimeRemaining = 5
+    var workPeriodLength = 10
+    var breakPeriodLength = 5
+    var timeToWork = true
+    
+    private lazy var timerLabel: UILabel = {
+        let label = UILabel()
+        label.text = (convertSecondsToTime(timeInSecomds: timeToShow))
+        label.font = .systemFont(ofSize: 40)
+        label.textColor = UIColor(named: "textColor")
+        return label
+    }()
+    
+    private lazy var startButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle("", for: .normal)
+        button.setImage(UIImage(systemName: "play.fill"), for: .normal)
+        button.addTarget(self, action: #selector(startButtonTapped), for: .touchUpInside)
+//        button.setImage(UIImage(systemName: "pause.fill"), for: .selected)
+        button.tintColor = UIColor(named: "buttonColor")
+        return button
+    }()
+    
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupHierarchy()
+        setupLayout()
+    }
+    
+    // MARK: - Setup
+    
+    private func setupHierarchy() {
+        
+        view.backgroundColor = UIColor(named: "ViewColor")
+        view.addSubview(timerLabel)
+        view.addSubview(startButton)
+        
+    }
+    
+    private func setupLayout() {
+        
+        timerLabel.snp.makeConstraints { make in
+            
+            make.centerX.equalTo(view)
+            make.centerY.equalTo(view)
```

Comment:
Тут можно вот так написать - более читаемо
```suggestion
            make.centerX.equalToSuperview()
            make.centerY.equalToSuperview()
```